### PR TITLE
BaSP-48: Timesheets redux structure

### PR DESCRIPTION
**Pull request by DinoStampella**

### CODE REVIEW

File: src/redux/projects/reducer.js [hunk 1]
```diff
@@ -1,5 +1,61 @@
-const projectsReducer = (state = null) => {
-  return state;
+import {
+  GET_PROJECTS_PENDING,
+  GET_PROJECTS_SUCCESS,
+  GET_PROJECTS_ERROR,
+  // PUT_PROJECTS_PENDING,
```
Comment (MatiasPazConti) [inline src/redux/projects/reducer.js:5]: Fix this comments

**Review by MatiasPazConti — CHANGES_REQUESTED** [review]
There are merging conflicts you must solve first

**Review by JulianFloresDev — DISMISSED** [review]
Approved

**Review by JulianFloresDev — APPROVED** [review]
Approve Changes

**Review by anapaulib — APPROVED** [review]
Nice

**Review by MatiasPazConti — APPROVED** [review]

**Pull request merged**